target/linux/ramips/dts/AFOUNDRY-EW1200.dts

```diff
@@ -0,0 +1,126 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/input/input.h>
```

Comment:
Please include  <[dt-bindings/gpio/gpio.h](http://lxr.free-electrons.com/source/include/dt-bindings/gpio/gpio.h?v=4.4)> here as well and keep alphabetical order..

Use the GPIO_ACTIVE_LOW and GPIO_ACTIVE_HIGH macros afterwards in stead of 1 and 0 in the gpio parameters.

Check the recent [ramips board additions](https://github.com/lede-project/source/commits/master/target/linux/ramips) for examples.